notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<p>I don't see QPS used anywhere above.. We should define it the first time we use it, just to be more straightforward for users. </p><p> </p><blockquote>Let's see how QPS (queries per second) depends on <code>k</code></blockquote><p><code>There's a couple typos in the original sentence too. </code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>